jso/apis/src/main/java/org/teavm/jso/indexeddb/IDBObjectStore.java

```diff
@@ -36,7 +36,7 @@
         }
     }
 
-    @JSBody(script = "return this;")
+    @JSBody(params = "obj", script = "return obj;")
     private native String[] unwrapStringArray(JSObject obj);
```

Comment:
I think this method should also be marked as 'static'

Reply:
Oh I didn't catch that, thanks for pointing it out